zebra-consensus/src/error.rs

```diff
@@ -177,6 +177,9 @@ pub enum TransactionError {
 
     #[error("must have at least one active orchard flag")]
     NotEnoughFlags,
+
+    #[error("could not find an input UTXO in best chain")]
+    InputNotFound,
```

Comment:
Nitpick: We already use `NoInputs` and `NoOutputs` to mean "shielded or transparent". So we might want to use a more specific name here.

```suggestion
    #[error("could not find a mempool transaction input UTXO in the best chain")]
    TransparentInputNotFound,
```